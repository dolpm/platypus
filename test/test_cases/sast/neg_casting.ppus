pipe main [] |> unit {
  int x = -5;
  int y = Float <| [x-500];
  
  |> ();
}
