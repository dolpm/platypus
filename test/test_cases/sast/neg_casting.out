Fatal error: exception Failure("illegal assignment int = float in int y = Float <| [x - 500];\n")
